shared/src/store/action-reducers/simulated-region.ts

```diff
@@ -268,4 +280,24 @@ export namespace SimulatedRegionActionReducers {
             },
             rights: 'participant',
         };
+
+    export const removeBehaviorToSimulatedRegion: ActionReducer<RemoveBehaviorFromSimulatedRegionAction> =
+        {
+            action: RemoveBehaviorFromSimulatedRegionAction,
+            reducer: (draftState, { simulatedRegionId, behaviorState }) => {
+                const simulatedRegion = getElement(
+                    draftState,
+                    'simulatedRegion',
+                    simulatedRegionId
+                );
+                simulatedRegion.behaviors.splice(
+                    simulatedRegion.behaviors.indexOf(
+                        cloneDeepMutable(behaviorState)
+                    ),
```

Comment:
This will always be `-1` because `indexOf` will use strict equality, and objects are compared by reference.

Reply:
Since this now uses the Id, this has been fixed